libvast/src/system/configuration.cpp

```diff
@@ -144,18 +155,24 @@ caf::error configuration::parse(int argc, char** argv) {
         caf::put(content, "vast.plugin-dirs", std::move(cli_plugin_dirs));
     }
   }
-  // Move CAF options to the end of the command line, parse them, and then
-  // remove them.
+  // Separate CAF options from the command line; we'll parse them later.
   auto is_vast_opt = [](const auto& x) {
     return !x.starts_with("--caf.");
   };
   auto caf_opt = std::stable_partition(command_line.begin(), command_line.end(),
                                        is_vast_opt);
   std::vector<std::string> caf_args;
+  // Prepopulate CAF args with environment variables so that the CLI can
+  // override them.
+  for (const auto& [key, value] : detail::environment())
+    if (key.starts_with("CAF_") && !value.empty())
```

Comment:
We should use `VAST_CAF__` as a prefix here to be consistent. VAST should not read environment variables that do not start with `VAST_`.

Reply:
Good point; addressed.